stylus/base/defaults.styl

```diff
@@ -9,3 +9,13 @@ $error
     border-radius   4px
     background-color red
     color            white
+.coz-hidden
```

Comment:
Maybe we should don't use dot-names, as they'll be exported in final CSS as classes, and stuck on dollar-names, that can be sfaely extended w/o infating the output. Except if we prefer to have a `.coz-hidden` class available, obviously (but I don't think it's a good idea, or we must document it very well)